include/sys/fs/zfs.h

```diff
@@ -799,13 +799,15 @@ typedef struct pool_scan_stat {
 	uint64_t	pss_start_time;	/* scan start time */
 	uint64_t	pss_end_time;	/* scan end time */
 	uint64_t	pss_to_examine;	/* total bytes to scan */
-	uint64_t	pss_examined;	/* total examined bytes	*/
+	uint64_t	pss_examined;	/* total bytes located by scanner */
+	uint64_t	pss_issued;	/* total bytes checked by scanner */
```

Comment:
Kernel Binary Interface.  As I see, this structure is passed from kernel to user-level as array of uint64_t values.  I bet that addition of value into the middle will break binary compatibility.